README.md

```diff
@@ -1,92 +1,123 @@
-    + [Doing the load](#doing-the-load)
-  * [Example 2a: Remote file to DataStore](#example-2a-remote-file-to-datastore)
-  * [Examples 3: Auto Load file uploaded to CKAN into CKAN DataStore](#examples-3-auto-load-file-uploaded-to-ckan-into-ckan-datastore)
-    + [Run it](#run-it)
-- [Tutorials](#tutorials)
-  * [Using Google Cloud Composer](#using-google-cloud-composer)
+- [💨🥫 AirCan](#-aircan)
+  - [Get Started](#get-started)
+  - [Examples](#examples)
+    - [Example 1: CSV to JSON](#example-1-csv-to-json)
+  - [Using Aircan DAGs](#using-aircan-dags)
+    - [Example 2: Local file to CKAN DataStore using the Datastore API](#example-2-local-file-to-ckan-datastore-using-the-datastore-api)
+      - [Preliminaries: Setup your CKAN instance](#preliminaries-setup-your-ckan-instance)
+    - [Example 3: Local file to CKAN DataStore using Postgres](#example-3-local-file-to-ckan-datastore-using-postgres)
+      - [Preliminaries: Setup your CKAN instance](#preliminaries-setup-your-ckan-instance-1)
+      - [Doing the load](#doing-the-load)
+    - [Example 2a: Remote file to DataStore](#example-2a-remote-file-to-datastore)
+    - [Examples 3: Auto Load file uploaded to CKAN into CKAN DataStore](#examples-3-auto-load-file-uploaded-to-ckan-into-ckan-datastore)
+      - [Run it](#run-it)
+  - [Tutorials](#tutorials)
+    - [Using Google Cloud Composer](#using-google-cloud-composer)
 
 <!-- tocstop -->
 
 ## Get Started
 
-* Install Python >= 3.xx (and make a virtualenv)
-* Clone aircan so you have the examples
-  ```
+* Install **Python** >= 3.5 (and make a virtual environment).
+* Clone `aircan` so you have examples available:
+
+  ```bash
   git clone https://github.com/datopian/aircan
   ```
-* Airflow: install and setup https://airflow.apache.org/docs/stable/installation.html
-  ```
+
+* Install and setup **Airflow** (https://airflow.apache.org/docs/stable/installation.html):
+
+  ```bash
   export AIRFLOW_HOME=~/airflow
   pip install apache-airflow
   airflow initdb
   ```
-* Start the server and visit your airflow admin UI
-  ```
+  
+_Note_: On recent versions of Python (3.7+), you may face the following error when executing `airflow initdb`:
+
+```bash
+ModuleNotFoundError: No module named 'typing_extensions'
+```
+
+This can be solved with `pip install typing_extensions`.
+
+* Then, start the server and visit your Airflow admin UI:
+
+  ```bash
   airflow webserver -p 8080
   ```
 
-Now jump into one of the examples.
+By default, the server will be accessible at `http://0.0.0.0:8080/` as shown in the output of the terminal where you ran the previous command.
 
+Now, jump into one of the examples.
 
 ## Examples
 
 ### Example 1: CSV to JSON
 
-In this example we'll run an aircan example to convert a CSV to JSON.
+In this example we'll run an AirCan example to convert a CSV to JSON.
 
-Create the DAG for loading
+Add the DAG to the default directory for Airflow to recognize it:
 
+```bash
```

Comment:
I have updated this example so that it can run locally straight from the files in the repo.